fast_response/MonitoringAndMocks/Data_Display.py

```diff
@@ -790,18 +427,4 @@ def make_bg_pval_dist(fontsize=15, lower_y_bound=-3.5, load_all = False):
     plt.savefig("mock_pvals.png")
```

Comment:
this should also be a path to your home directory, right?

Reply:
oops, missed the line above. it is both. you can remove this line in favor of the one above